templates/base.html

```diff
@@ -38,7 +38,8 @@
     <link rel="icon" type="image/png" href="{{ url_for('static', filename='img/favicon/favicon-16x16.png') }}" sizes="16x16">
     <link rel="manifest" href="{{ url_for('static', filename='img/favicon/manifest.json') }}">
     <link rel="mask-icon" href="{{ url_for('static', filename='img/favicon/safari-pinned-tab.svg') }}" color="#5bbad5">
-    <link rel="shortcut icon" href="{{ url_for('static', filename='img/favicon/favicon.ico') }}">
+    <!-- favicon changes depending on server type -->
+    <link rel="shortcut icon" href="{{ url_for('static', filename=config['FAVICON']) }}">
```

Comment:
Can this be revised to `config.get('FAVICON', 'favicon'ico')` in order to provide a default value? Thank you.